docs/index.rst

```diff
@@ -1,3 +1,8 @@
+.. arrow documentation master file, created by
```

Comment:
Let's remove this comment to keep this file clean :).

Reply:
Done.